doc/ws.md

```diff
@@ -0,0 +1,121 @@
+# Websockets interface
+
+## What is this?
+
+This interface is a new way to talk with Orion using websockets, this is point to improve the comunication with a faster and efficient protocol, now we have a persistent connection with a open socket. 
+
+This feature is in a very early phase, at this moment you can create, delete, modify and update entities and you can subscribe for notifications, but is not well tested and up to date is only a wrapped version of REST API, there is not a binary protocol.
+
+## How I can use it?
+
+In the websocket protocol we need wrap some parts of REST API in JSON format in order to get a complete websocket message, to do this
```

Comment:
we need **to** wrap some parts of  **the** REST API ... complete websocket message. **To** do this ...